test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -198,7 +199,7 @@ def test_dependency_check_cpu(cpu, ec2_connection):
 
 @pytest.mark.model("N/A")
 @pytest.mark.parametrize("ec2_instance_type", ["p3.2xlarge"], indirect=True)
-@pytest.mark.skip(reason="Skipping due to bintray limit")
+# @pytest.mark.skipif(is_pr_context(), reason="Do not run dependency check on PR tests")
```

Comment:
Is it commented temporary? Will you uncomment it before merging?

Reply:
Yes. The skipif marker has been commented so that the test can run on PR context for this PR. I will uncomment it when the PR is ready to be merged, so that the test will only be run on Mainline and Nightly contexts.